src/controllers/auth.controller.js

```diff
@@ -15,8 +17,8 @@ class AuthController {
  * @static
  * @description POST /api/v1/signUp
  * @param {object} req request object
- * @param {object} res ResponseService object
- * @memberof AuthController
+ * @param {object} res response object
+ * @memberof UsersController
```

Comment:
Should be AuthController

Reply:
@victor-abz this has been already resolved. I have got it as feedback from @erickyvand .

Reply:
Line 21?

Reply:
Ok. worked on it. You can re review. I have finished your feedback.